pai-management/k8sPaiLibrary/maintaintool/kubectl-install.sh

```diff
@@ -19,12 +19,11 @@
 
 mkdir -p ~/.kube
 
-[[ -f "/usr/local/bin/kubectl" ]] &&
-{
+if which kubectl > /dev/null; then
```

Comment:
what's the difference between these two different style.

Reply:
I put kubectl in differnt location, but in PATH. Previous method failed in this case.